internal/host/installcmd.go

```diff
@@ -95,6 +95,11 @@ func (i *installCmd) GetStep(ctx context.Context, host *models.Host) (*models.St
 		}
 	}
 
+	if i.instructionConfig.ServiceCACert != "" {
```

Comment:
This is a string, right? What would this be set to? I would have expected the code below to use the value in `i.instructionConfig.ServiceCACert`.

If it's just a flag, can it be a bool called `ConfigureServiceCACert` or something?

Reply:
This is actually looking for i.instructionConfig.ServiceCACertPath. If that is set on the service, we want to have the instruction to have cacert set to HOST_CA_CERT_PATH . Hopefully, the variable renaming made it clearer.